CMakeLists.txt

```diff
@@ -287,6 +288,8 @@ set_target_properties(rocksdb${ARTIFACT_SUFFIX} PROPERTIES COMPILE_FLAGS "-DROCK
 add_dependencies(rocksdb${ARTIFACT_SUFFIX} GenerateBuildVersion)
 target_link_libraries(rocksdb${ARTIFACT_SUFFIX} ${LIBS})
 
+include(${CMAKE_CURRENT_SOURCE_DIR}/java/jni.inc)
```

Comment:
@koldat Pls, also make sure things work in release  build. So 4 permutations 1,2) Debug with and w/o Java 3,4) same with Release